core/services/relay/evm/functions/config_poller_test.go

```diff
@@ -203,3 +204,27 @@ func setFunctionsConfig(t *testing.T, pluginConfig *functionsConfig.ReportingPlu
 		OffchainConfig:        offchainConfig,
 	}
 }
+
+func generateDefaultOCR2OnchainConfig(minValue *big.Int, maxValue *big.Int) []byte {
```

Comment:
@samsondav any suggestions where we can put this utility function for building onchain configurations?

Reply:
If its used in tests it doesnt matter too much. You could put in `evmutils`